task/runner.go

```diff
@@ -671,10 +671,14 @@ func deleteAsset(asset *api.Asset, r *runner, ctx context.Context) error {
 			glog.Errorf("Error unpinning from IPFS %v", ipfs.CID)
 			return err
 		}
-		err = r.UnpinFromIpfs(ctx, ipfs.NFTMetadata.CID, "nftMetadataCid")
-		if err != nil {
-			glog.Errorf("Error unpinning metadata from IPFS %v", ipfs.NFTMetadata.CID)
-			return err
+		if ipfs.NFTMetadata != nil {
```

Comment:
Wow didnt remember we could have IPFS exports without NFT. Maybe some really old assets